package.json

```diff
@@ -233,6 +238,34 @@
 									}
 								}
 							},
+							"enforceNamingConvention":{
+								"type": "object",
+								"description": "enforce naming convention when converting file(s)",
+								"properties": {
+									"yaml": {
+										"type": "string",
+										"description": "yaml target naming convention",
+										"enum": [
+											"kebab-case",
+											"snake_case",
+											"camelCase",
+											"PascalCase",
+											"none"	
```

Comment:
I'm not sure if it really make sense to have `none`? 🤔 I would expect that to simply be equal to not setting this at all which I would also expect to be the default option.

Reply:
Hey @Droxx 

I didn't see any comments or changes here but you resolved the comment. Wondering what your conclusion was?

For what use case would you set this setting and define the config as `none`?

Reply:
@hilleer I removed it everywhere else. This occurance was left by mistake. Pushed a fix